src/authentication/githubAuthProvider.tsx

```diff
@@ -4,8 +4,9 @@ import BaseAuthProvider from './baseAuthProvider';
 import ReactGA from 'react-ga';
 
 export default class GithubAuthProvider extends BaseAuthProvider {
-  public constructor() {
-    super();
+  public constructor(authUrl: string | undefined) {
+    super(authUrl);
+    this.authUrl = authUrl;
```

Comment:
Surely calling `super(authUrl)` already sets `this.authUrl`, so this is superfluous?